cmd/snap/cmd_run.go

```diff
@@ -760,6 +760,31 @@ func snapdHelperPath(toolName string) (string, error) {
 	return filepath.Join(snapBase, dirs.CoreLibExecDir, toolName), nil
 }
 
+/* Kerberos tickets live in /tmp, or in the place specified by KRB5CCNAME
+ * environment variable. However, in snaps /tmp is private.
+ * So rewire the environment variable to point to /var/lib/snapd/hostfs/tmp,
+ * in case the variable is unset or resolves tickets to /tmp/krb5cc*.
+ *
+ * Snaps that want to read Kerberos tickets must then connect to the
+ * kerberos-tickets interface to have access to the tickets.
+ */
+func exposeKerberosTickets(info *snap.Info) (string, error) {
+	krb5EnvVar := osGetenv("KRB5CCNAME")
```

Comment:
Nitpick, can you either now or in a follow-up move this to package level const. The name is so peculiar and it is used more than once.